Marlin/src/gcode/calibrate/G28.cpp

```diff
@@ -145,10 +145,6 @@
         active_extruder_parked = false;
       #endif
 
-      #if ENABLED(SENSORLESS_HOMING)
-        safe_delay(500); // Short delay needed to settle
-      #endif
```

Comment:
@teemuatlut — Maybe this should be kept, but with `&& DISABLED(IMPROVE_HOMING_RELIABILITY)`…?

Reply:
I restored the delay.